import-recipes/recipes.json

```diff
@@ -1,4 +1,36 @@
 [
+  {
+    "alias": "Cantabular Example 1",
+    "format": "cantabular-table",
+    "_id": "38542eb6-d3a6-4cc4-ba3f-32b25f23223a",
+    "cantabular_blob": "Example",
+    "output_instances": [
+      {
+        "code_lists": [
+          {
+            "href": "",
```

Comment:
Will we need the code list hrefs? Maybe we could create them as:
```
http://localhost:22400/code-lists/<id>
```
(similar to other recipes)

Reply:
Had a discussion with Eleanor earlier about this, she suggested to leave them blank for now as we won't be needing them  yet or accessing them through those URLS, so something to come back to